vamp-plugins/maths/Polyfit.h

```diff
@@ -31,7 +31,7 @@
              specific number of points.
 2001 Jan 07  Update Web site address
 
-Copyright � David J Taylor, Edinburgh and others listed above
```

Comment:
Why did you replace this? This looks unrelated.

Reply:
Because it was a latin-1 encoded (c), It looks like � in eclipse. 
We should just avoid non ascii character in our source.

Reply:
Do we really need to edit external files without any real need?

Reply:
I have found it via a lind tool during packaging. This file is no longer extern in 2.3. but if you all prefer to revert it I can do. Should I?

Reply:
Let's revert those changes.